UserTools/Configuration/Configuration.cpp

```diff
@@ -72,15 +74,31 @@ void Configuration::Thread(Thread_args* arg){
 
 bool Configuration::LoadConfig(){
 
+  //m_data->services->SendLog("Loading configuration "+std::to_string(m_data->run_configuration)+" from DB" , 1);
+  Log("Loading configuration "+std::to_string(m_data->run_configuration)+" from DB" , 1);
   std::string config_json="";
+  std::string response="";
```

Comment:
cant see where this is used

Reply:
yep that's cruft, was added then code changed, no longer needed.